include/pocl.h

```diff
@@ -242,6 +242,7 @@ typedef struct _cl_command_node_struct
   const cl_event *event_wait_list;
   cl_uint num_events_in_wait_list;
   cl_device_id device;
+  cl_uint isPoclccBinary;
```

Comment:
This should probabnly not be here but only in program since it's possible to get the program from the command node.